models/agent_model.go

```diff
@@ -220,7 +220,7 @@ type DBConfig struct {
 
 // Valid returns true if config is valid.
 func (c DBConfig) Valid() bool {
-	return c.User != "" && (c.Address != "" || c.Socket != "")
+	return c.Address != "" || c.Socket != ""
```

Comment:
can be user unspecified?

Reply:
Yes, at least for `mongoDB`. For MySQL not sure, need to check.

Reply:
@Dasio regarding [xtrabackup](https://www.percona.com/doc/percona-xtrabackup/8.0/xtrabackup_bin/xbk_option_reference.html) and [mysql](https://linux.die.net/man/1/mysql) documentation user is optional, in case when it's not specified defaults used (root/emtpy pass)

Reply:
I tested it. Yes, user can be empty also for MySQL. So I made changes in MySQL backup job In pmm agent.